tekton/publish.yaml

```diff
@@ -111,13 +111,16 @@ spec:
       value: "off"
     - name: GOFLAGS
       value: "-mod=vendor"
-    - name: CLOUDSDK_AUTH_CREDENTIAL_FILE_OVERRIDE
-      value: /secret/release.json
+    - name: GOOGLE_APPLICATION_CREDENTIALS
+      value: "/secret/release.json"
     script: |
       #!/usr/bin/env bash
       set -ex
 
-      # Auth with CLOUDSDK_AUTH_CREDENTIAL_FILE_OVERRIDE
+      # Activate service account
+      gcloud auth activate-service-account --key-file=${GOOGLE_APPLICATION_CREDENTIALS}
+
+      # Setup docker-auth
       gcloud auth configure-docker
```

Comment:
NIT: we might want to add --quiet here so that it doesn't prompt for confirmation